HistoryQueryManager/src/main/java/eu/neclab/ngsildbroker/historyquerymanager/repository/HistoryDAO.java

```diff
@@ -117,7 +117,7 @@ public Uni<Table<String, String, RegistrationEntry>> getAllRegistries() {
 								while (it2.hasNext()) {
 									Row row = it2.next();
 									result.put(tenant, row.getString(1),
-											DBUtil.getRegistrationEntry(row, tenant, logger));
+											List.of(DBUtil.getRegistrationEntry(row, tenant, logger)));
```

Comment:
this overwrites previous entries with the new one and doesn't add to it
do something like this from entity manager
return Uni.combine().all().unis(unis).combinedWith(list -> {
							Table<String, String, List<RegistrationEntry>> result = HashBasedTable.create();
							for (Object obj : list) {
								Tuple2<String, RowSet<Row>> tuple = (Tuple2<String, RowSet<Row>>) obj;
								String tenant = tuple.getItem1();
								RowIterator<Row> it2 = tuple.getItem2().iterator();
								while (it2.hasNext()) {
									Row row = it2.next();
									List<RegistrationEntry> entries = result.get(tenant, row.getString(1));
									if (entries == null) {
										entries = Lists.newArrayList();
										result.put(tenant, row.getString(1), entries);
									}
									entries.add(DBUtil.getRegistrationEntry(row, tenant, logger));
								}
							}
							return result;
						});

Reply:
done